.tool-versions

```diff
@@ -1,3 +1,3 @@
-elixir 1.17.1-otp-27
-erlang 27.0
+elixir 1.17.2-otp-27
```

Comment:
If you're going to change the versions in this place, you need to do the same in CI, Dockerfile and everywhere else they are referenced. Either revert or fix the version in these other places too.